package-lock.json

```diff
@@ -5323,7 +5332,8 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true
+          "bundled": true,
+          "optional": true
```

Comment:
I can't realize why to change this..... 🤔 

My Node.js environment is described in the following:
```
[I] ~/.g/g/c/crowi > node --version                                       master
v10.16.0
[I] ~/.g/g/c/crowi > npm --version                                        master
6.9.0
```

Reply:
package-lock.json in master is old, so your change is correct.

Reply:
@otofune But Crowi `v1.7.x` supports Node.js v8. And Node.js v8.16.0 uses npm 6.4.0.
Can you confirm the version?
(We will move Node.js v10 with Crowi v1.8)

Reply:
!! 
I'll re-generate package.json with node v8 environment.